app/helpers/project_builder.rb

```diff
@@ -0,0 +1,8 @@
+module ProjectBuilder
```

Comment:
My personal experience is that these techniques using modules to share behavior is hard to maintain/extend. They can quickly grow out of control, start having dependencies between modules and bidirectional access with the classes that includes this modules. Also it's hard to follow the code since you can call methods without being explicit on where that method is defined. My recommendation is just using other POROs if you want to encapsulate behavior